docs/users-guide-docs/documentation/earthportal/about.md

```diff
@@ -5,5 +5,5 @@ permalink: user_guide/EarthPortal
 parent: Users guide
 nav_order: 1
 ---
-{% include documentation_nav.html portal="EarthPortal" %}
+{% include documentation_nav.html %}
```

Comment:
For the nav, when you do the include you need to explicitly specify the portal (this is why it's not getting colored when you select it),
The config you added do the job only for the paths where we have the shared doc, but for the nav you can't do the same thing